### PR TITLE
fix grammatical errors in documentation

### PR DESCRIPTION
**Pull request by Shivam175**
### Description
Fixed  grammatical  errors in the documentation, PR not under any specific issue.
Changes in 2 files: 
1. docs/graphql/manual/auth/authentication/index.rst
2. docs/graphql/manual/auth/authentication/jwt.rst

### Affected components 

- Docs

### Related Issues
--

### Solution and Design
--

### Steps to test and verify
--

### Limitations, known bugs & workarounds
--

### CODE REVIEW

File: docs/graphql/manual/auth/authentication/index.rst [hunk 1]
```diff
@@ -33,7 +33,7 @@ Hasura supports two modes of authentication configuration:
 
    .. thumbnail:: ../../../../img/graphql/manual/auth/auth-jwt-overview.png
 
-**See more details about these at:**
+**See more details about these, at:**
```
Comment (marionschleifer) [inline docs/graphql/manual/auth/authentication/index.rst]: Could you revert this change please? This doesn't read nicely.

**Comment by Shivam175** [comment]
Sorry, clicked on merge by mistake, i'll make the changes right away.

**Comment by marionschleifer** [comment]
@Shivam175 could you please send an email to marion@hasura.io, so that we can arrange the shipping of your swag? 🙂

**Comment by Shivam175** [comment]
Yup, will send it right now. Thanks guys! 😃
